src/components/ExperienceList.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import ExperienceCard from './ExperienceCard';
+
+const ExperienceList = () => {
+  const experience = [
```

Comment:
can you move this experience to a json or js file ourside the component?

Reply:
done